### PR TITLE
add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by basitali111**
Learning objectives
- Implement classes and objects in Ruby.
- Implement encapsulation and inheritance with Ruby.
- Run a program using the command line.

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @basitali111,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### 1. Kindly format your README to be more professional using the Microverse template
### 2. Kindly set up your linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,75 +1,38 @@
...
+Hat tip to anyone whose code was used
+Inspiration
+etc
 ## 📝 License
+This project is MIT licensed.
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```
Comment (rustam-tolipov) [inline README.md:38]: [OPTIONAL] Kindly Add MIT license to your project.

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```
Comment (rustam-tolipov) [inline person.rb:22]: [OPTIONAL] You can remove `public` and put your `can_use_services?` method before private. Then, it can be used as public.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @basitali111👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

No Linter errors 💯

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**